runtime/common/src/main/kotlin/kotlinx/serialization/json/JsonElement.kt

```diff
@@ -146,25 +146,12 @@ public sealed class JsonPrimitive : JsonElement() {
  * and `JsonLiteral("42", true)` will be serialized as `42` and `"42"` respectively.
  */
 @Serializable(JsonLiteralSerializer::class) // TODO val for body is workaround for plugin bug
-public class JsonLiteral internal constructor(val body: Any, public val isString: Boolean) : JsonPrimitive() {
+sealed class JsonLiteral<T> private constructor(val body: T) : JsonPrimitive() {
+    @Deprecated("Check types instead")
+    public abstract val isString: Boolean
```

Comment:
This property was not released, so you can safely remove it. 
Also, it should not be used in the implementation if it is deprecated

Reply:
it is `public` in master now